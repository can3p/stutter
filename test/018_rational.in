(list
  1/2
  (+ 1/2 0.2)
  (+ 1/2 1)
  (+ 1 1/2)
  (+ 1/3 1/6)
  -1/2
  (- 1/2)
  (- -1/2)
  (- 1/2 1/3)
  (- 1 1/2)
  (- 1/2 0.25)
  (* 1/2 3)
  (* 1/2 1/3)
  (* 3/2 0.5)
  (/ 2)
  (/ 1/2 1/3)
  (/ 1/2 3))
