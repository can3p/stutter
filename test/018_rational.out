(#t #f #t #t #f #f 1/2 0.7000000000 3/2 3/2 1/2 -1/2 -1/2 1/2 1/6 1/2 0.2500000000 3/2 1/6 0.7500000000 1/2 3/2 1/6)
